app/code/core/Mage/Core/Model/Resource/File/Storage/File.php

```diff
@@ -214,6 +208,6 @@ public function saveFile($filePath, $content, $overwrite = false)
             Mage::throwException(Mage::helper('core')->__('Unable to save file: %s', $filePath));
         }
 
-        return false;
+        return true;
```

Comment:
he said, there is not usage of the return value and it seems wrong to him, returning false when overwrite-argument is false. IMHO it should stay "return false" because 1) no usages in core does not mean no usages in extensions and 2) if I call this method to save content to a file, I don't want it to return true if nothing was written (bcause file already existed and I did not provide the overwrite-flag).